src/app.js

```diff
@@ -49,53 +49,11 @@ p5.prototype.setVRBackgroundColor = function(r, g, b) {
   p5xr.instance.curClearColor = color(r, g, b);
 };
 
-p5.RendererGL.prototype._update = function() {
-  /* TODO: Make a diff function to avoid overwriting this function */
-  /* IE: Override the resetting of cameraMatrices in _update */
-  /*
-    /*
-    /*
-    /*
-    /*
-    */
-
-  // reset model view and apply initial camera transform
-  // (containing only look at info; no projection).
-  // this.uMVMatrix.set(
-  //   this._curCamera.cameraMatrix.mat4[0],
-  //   this._curCamera.cameraMatrix.mat4[1],
-  //   this._curCamera.cameraMatrix.mat4[2],
-  //   this._curCamera.cameraMatrix.mat4[3],
-  //   this._curCamera.cameraMatrix.mat4[4],
-  //   this._curCamera.cameraMatrix.mat4[5],
-  //   this._curCamera.cameraMatrix.mat4[6],
-  //   this._curCamera.cameraMatrix.mat4[7],
-  //   this._curCamera.cameraMatrix.mat4[8],
-  //   this._curCamera.cameraMatrix.mat4[9],
-  //   this._curCamera.cameraMatrix.mat4[10],
-  //   this._curCamera.cameraMatrix.mat4[11],
-  //   this._curCamera.cameraMatrix.mat4[12],
-  //   this._curCamera.cameraMatrix.mat4[13],
-  //   this._curCamera.cameraMatrix.mat4[14],
-  //   this._curCamera.cameraMatrix.mat4[15]
-  // );
-  // p5.instance._renderer.uMVMatrix.set(viewMat);
-  // p5.instance._renderer.uPMatrix.set(projMat);
-  // reset light data for new frame.
-
-  /*
-    /*
-    /*
-    /*
-    /*
-    */
-  /* TODO: Make different function to avoid overwriting this function */
-  /* IE: Override the resetting of cameraMatrices in _update */
-
+p5.RendererGL.prototype._updatexr = function () {
```

Comment:
Does it make sense to keep this as a p5.RendererGL prototype? It might offer some advantage down the road but it could also easily be a p5xr method since it is "private" and it is assumed that the user never calls it. This might be a good way to distinguish what should be exposed to the user.